src/types.ts

```diff
@@ -275,7 +283,15 @@ export interface DecideResponse {
     featureFlagPayloads: Record<string, JsonType>
     errorsWhileComputingFlags: boolean
     autocapture_opt_out?: boolean
-    capturePerformance?: boolean
+    /**
+     *     originally capturePerformance was replay only and so boolean true
+     *     is equivalent to { network_timing: true }
+     *     now capture performance can be separately enabled within replay
+     *     and as a standalone web vitals tracker
+     *     people can have them enabled separately
+     *     they work standalone but enhance each other
+     */
+    capturePerformance?: boolean | PerformanceCaptureConfig
```

Comment:
it makes more sense for `capturePerformance: true` to mean `{ network_timing: true, web_vitals: true}`

but then we're turning this on for anyone that has `capturePerformance: true` (should not be many people 🤷)

but I decided to not be too YOLO here